milli/src/search/mod.rs

```diff
@@ -18,7 +18,6 @@ pub(crate) use self::facet::ParserRule;
 pub use self::facet::{FacetDistribution, FacetNumberIter, FilterCondition, Operator};
 pub use self::matching_words::MatchingWords;
 use self::query_tree::QueryTreeBuilder;
-use crate::error::FieldIdMapMissingEntry;
```

Comment:
@MarinPostma  Is this error used anywhere else now? Can't we just remove it?

Reply:
this is used when building documents in obkv_json, at least